### ISSUE TITLE
F18 Presets not consistent

### ISSUE THREAD

**Issue by martinco** [132nd-vWing/ATRM#96]
I was pulling radio presets for MDC from the ATRM mission file itself (to validate) and came across 4 FA 18Cs that had different presets to the others:

```
#!/usr/bin/lua

-- Source mission table
dofile "mission"

-- https://raw.githubusercontent.com/rxi/json.lua/master/json.lua
local json = require "json"

output = {}

for cid,cdata in pairs(mission['coalition']['blue']['country']) do
  for ce_elem, ce_data in pairs(cdata) do
    if ce_elem == "plane" or ce_elem == "helicopter" then
      for gid, gdata in pairs(ce_data['group']) do
        for u_id, u_data in pairs(gdata['units']) do
          if u_data['skill'] == 'Client' then
            if u_data['type'] == 'FA-18C_hornet' then

              local g_n = gdata['name']
              local u_n = u_data['name']

              local radios = {}

              for radio_id, radio_data in pairs(u_data['Radio']) do

                radios[radio_id] = {}

                for k, v in pairs(radio_data['channels']) do
                  radios[radio_id][k] = v
                end
              end


              output[g_n .. "-" .. u_n] = json.encode(radios)
            end
          end
        end
      end
    end
  end
end

local maps = {}
for id, radios in pairs(output) do
  if maps[radios] == nil then
    maps[radios] = {}
  end

  table.insert(maps[radios], id)
end

print(json.encode(maps))
```

The output being:

```
{
  "[[230.25,234.5,245.25,239,227.75,149,148,229,242.5,239.75,243.75,225.75,235.75,233.25,234.75,230.5,236.5,245,228.75,243],[126.1,126.2,127.1,237,228,248,235,246.25,230,309,309.5,309.9,230.25,234.5,245.25,239,227.75,234,228.25,243]]": [
    "DictKey_GroupName_6222-DictKey_UnitName_6223",
    "DictKey_GroupName_6237-DictKey_UnitName_6238",
    "DictKey_GroupName_6234-DictKey_UnitName_6235",
    "DictKey_GroupName_6155-DictKey_UnitName_6156"
  ],
  "[[230.25,234.5,245.5,239,227.75,149,148,249,242.5,262,239.75,243.75,225.75,233.25,234.75,230.5,236.5,245,228.75,243],[126.1,126.2,127.1,237,228,248,235,246.25,230,309,309.5,309.9,230.25,234.5,242.25,239,227.75,234,228.25,243]]": [
    "DictKey_GroupName_1890-DictKey_UnitName_1893",
    "DictKey_GroupName_1878-DictKey_UnitName_1879",
    "DictKey_GroupName_1890-DictKey_UnitName_1894",
    "DictKey_GroupName_1884-DictKey_UnitName_1888",
    "DictKey_GroupName_558-DictKey_UnitName_1877",
    "DictKey_GroupName_1878-DictKey_UnitName_1881",
    "DictKey_GroupName_558-DictKey_UnitName_1875",
    "DictKey_GroupName_558-DictKey_UnitName_559",
    "DictKey_GroupName_1884-DictKey_UnitName_1885",
    "DictKey_GroupName_3257-DictKey_UnitName_3260",
    "DictKey_GroupName_3257-DictKey_UnitName_3258",
    "DictKey_GroupName_1878-DictKey_UnitName_1882",
    "DictKey_GroupName_558-DictKey_UnitName_1876",
    "DictKey_GroupName_3257-DictKey_UnitName_3261",
    "DictKey_GroupName_3257-DictKey_UnitName_3259",
    "DictKey_GroupName_1878-DictKey_UnitName_1880",
    "DictKey_GroupName_1890-DictKey_UnitName_1891",
    "DictKey_GroupName_1890-DictKey_UnitName_1892",
    "DictKey_GroupName_1884-DictKey_UnitName_1887",
    "DictKey_GroupName_1884-DictKey_UnitName_1886"
  ]
}
```

The top 4 being (as displayed)

- F/A-18 #6-3 park Stennis - F/A-18 #6-3
- F/A-18 #6-4 park Stennis - F/A-18 #6-4
- F/A-18 #6-2 park Stennis - F/A-18 #6-2
- F/A-18 #6-1 park Stennis - F/A-18 #6-1

Neither set match the ATRM Frequency and Callsign master list for F/A-18C Presets as follows:

Top 4:
  COMM 1 Preset 8 - 229 instead of 249

Bottom Group:

  COMM 1 Preset 3   - 245.5 instead of 245.25
  COMM 1 Presets 10 to 13 are off due to inclusion of 262 at 10, and absence of 235.75 at 13, so need shifting back by 1 and 235.75 added

  COMM 2 Preset 15 - 242.25 instead of 245.25

Given the closeness of the presets of the 4 Stennis F/A-18's to the Master sheet, it appears that the master sheet is (as hoped) correct.

If so, i'm happy to submit a PR to make these all consistent :)

**Comment by 132ndNeck** [comment]
Very happy with a PR to make all consistent if that is easy to do.
Different people had added F/A-18s at various stages in the .miz development, and can be cause for the inconsistency of the presets.

**Comment by martinco** [comment]
No worries, In that case, i'll submit a PR when I get a chance (this evening or tomorrow) to:

  - make the 4 Stennis correct as per master sheet (Change preset 8 to 249) 
  - Update the remaining 18s to match

Thanks,
Martin

**Comment by martinco** [comment]
I'm just holding for the moment for any pending mergers of any SC related things / R15 items from Junior / Jaeger to avoid any rebasing for folks :)

**Comment by 132ndNeck** [comment]
One thing though, I am not sure just making a PR here works. I think this is the reason why we use etcher's EMFT for the collabarative work on the .miz.

@martinco I am very happy to go through how it is done (and with your experience, it should be super easy)

But maybe @theendsofinvention can fill in here if one can simply make a PR, or if one should use the EMFT to not corrupt the .miz in any way?

**Comment by martinco** [comment]
#105 merged

**Comment by 132ndNeck** [comment]
@martinco @theendsofinvention @132nd-etcher 
Just doing some housekeeping, is this one fixed? (I think we fixed this before summer, but now I am not sure, since I had not closed this issue).

**Comment by 132nd-etcher** [comment]
This was solved in #105

**Comment by 132nd-etcher** [comment]
Closing: 132ndNeck merged commit d94b46c into 132nd-vWing:master on Jun 8

**Comment by 132ndNeck** [comment]
Thanks, that is what I thought :)
